website/docs/contributing.md

```diff
@@ -125,15 +125,15 @@ This will invoke the Typescript compiler (tsc) to compile the type definitions (
 
 This will run both the `build` script and `build:type` one.
 
-#### `yarn fmt`
+#### `yarn check`
 
-This will format your code to match the rest of the codebase.
+This will do all the checking, linting and formatting to make sure your code follows the linting and formatting rules in place.
 
-This will run every time you make a commit. If you disable this you should still run it manually before creating the pr
+Every time you make a commit a check will also run. If you disable this you should still run it manually before creating the pr
 
-#### `yarn lint`
+#### `yarn check:all`
 
-This will do all the checking and linting to make sure your code is in a useable state. There is also one other key thing to note about this. When you change something in the `@discordeno/types` package, you can run `yarn lint` to rebuild all the types.
+This is a script that only exists in the root of the repository, it will check all the files in repo including examples, website, etc
```

Comment:
```suggestion
This is a script that only exists in the root of the repository. It will check all the files in repo including examples, website, etc
```